locale.go

```diff
@@ -8,11 +8,11 @@ import (
 
 // Detect will detect current env's language.
 func Detect() (tag language.Tag, err error) {
-	tags, err := DetectAll()
+	lang, err := detect()
 	if err != nil {
 		return language.Und, err
 	}
-	return tags[0], nil
+	return language.Make(lang[0]), nil
```

Comment:
`return language.Parse(lang[0])` here might make sense, but that would return additional errors that are ignored today, resulting in a potentially breaking change.